app/views/planning_application_mailer/validation_notice_mail.text.erb

```diff
@@ -1,45 +1,17 @@
-
-We aim to issue a decision by <%= @planning_application.expiry_date %>. However, if your application has not been determined by <%= @planning_application.expiry_date %>, you have the right to appeal to the Secretary of State, either online at https://www.gov.uk/government/organisations/planning-inspectorate, or by post to Temple Quay House, 2 The Square, Temple Quay, Bristol BS1 6PN.
-
-An appeal in this situation assumes the refusal of the application, even if it had intended to be granted. It is therefore recommended that you consult your case officer before taking such action.
-
-If you wish to appeal, use the Planning Inspectorate’s online appeals service. To find out more, follow the link below. The Planning Inspectorate will publish your appeal details on its website, including the documents you submitted as part of your planning application, along with your completed appeal form and any other information required. Ensure any personal information provided belongs to you and that its publication is not an issue. If you provide information belonging to someone else make sure you have their permission.
-
-Signed: <%= @planning_application.local_authority.signatory_name %>
-<%= @planning_application.local_authority.signatory_job_title %>
-
-Your attention is drawn to the notes accompanying this document
+Application reference number: <%= @planning_application.reference_in_full %>
 
-Any enquiries regarding this document should quote the Application Number and be sent to
-the
-<%= @planning_application.local_authority.signatory_job_title %>,
-<%= @planning_application.local_authority.name %>,
-<%= @planning_application.local_authority.enquiries_paragraph %>
-or by email to <%= @planning_application.local_authority.email_address %>
+Address: <%= @planning_application.full_address.upcase %>
 
-Your proposal may also require approval under the Building Regulations and you should contact our Building Control team.
+Your application for a Lawful Development Certificate contains no errors or missing information. We can now start assessing your application against the relevant planning policies and legislation.
 
-A Community Infrastructure Levy (CIL) charge may be applicable to your application. Further information is available here: https://www.gov.uk/guidance/community-infrastructure-levy
+We aim to reach a decision about your application by <%= @planning_application.expiry_date.strftime("%-d %B %Y") %>. If we need more time, we will contact you. If you have not heard back from us by then, you have the right to appeal to the Secretary of State.
 
-Please Note: In light of the COVID-19 situation, we are following the current advice from the government to minimise travelling and non-essential contact.
+Find out [how to appeal an application for a Lawful Development Certificate](https://www.gov.uk/appeal-lawful-development-certificate-decision).
```

Comment:
Turns out we don't have html templates for emails because we're using the [mail-notify](https://github.com/dxw/mail-notify) gem. The `view_mail` method requires text templates, but also supports the use of markdown.

Reply:
Ah good to know